src/bytes.rs

```diff
@@ -1259,6 +1259,17 @@ mod tests {
         assert!(re.is_match(b("a")).unwrap());
     }
 
+    #[test]
```

Comment:
Sorry to keep adding on here, but can you put a comment indicating that this is a regression test, along with a link to #10?

Reply:
No problem! Let me know if you want the comment changed.